frontend/user/routes.py

```diff
@@ -15,10 +16,22 @@
 @user_bp.route("")
 @login_requested
 def user():
```

Comment:
May be good to have a docstring in here now that it's a bit dynamic?

Reply:
added docstrings! :D